server/container/guice/cassandra-rabbitmq-guice/src/test/java/org/apache/james/CassandraRabbitMQAwsS3JmapTestRule.java

```diff
@@ -68,11 +69,9 @@ public GuiceJamesServer jmapServer(Module... additionals) throws IOException {
             .build();
 
         return GuiceJamesServer.forConfiguration(configuration)
-            .combineWith(CassandraRabbitMQJamesServerMain.MODULES)
+            .combineWith(CassandraRabbitMQJamesServerMain.baseModule(objectStorage()))
```

Comment:
please avoid static import, it adds context to the reader to read `BlobStoreConfiguration.objectStorage()`